mirrord-agent/src/sniffer.rs

```diff
@@ -470,6 +468,9 @@ impl TCPConnectionSniffer {
             Some(session) => session,
             None => {
                 if !is_new_connection(tcp_flags) {
+                    if tcp_flags == 24 {
+                        debug!("handle_packet -> wiwiwi {:?}", &tcp_packet);
```

Comment:
Is the wiwiwi intentional?

Reply:
it's a temp log until we find the issue :D